cloud/helm-chart/src/main/helm/zilla/templates/ingress.yaml

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2021-2023 Aklivity Inc
+#
+# Licensed under the Aklivity Community License (the "License"); you may not use
+# this file except in compliance with the License.  You may obtain a copy of the
+# License at
+#
+#   https://www.aklivity.io/aklivity-community-license/
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations under the License.
+#
```

Comment:
Most helm charts don't include license text in the templates directory since these files are to be parsed by the helm template parser. If we do need to add these then we need to use the proper [comment format](https://github.com/bitnami/charts/blob/main/bitnami/kafka/templates/secrets.yaml#L1).

Reply:
@jfallows, can we remove the license text from everything under `templates/`? The license will still exist in the `Chart.yaml`, meaning when Helm makes the full deployment artifact, it will have the license comment in it.

Reply:
I believe `#` comments are valid yaml. Including or excluding the license text is a question to @jfallows, I guess.

Reply:
Correct, the `Chart.yaml` file is a valid yaml file, but everything in the `templates` folder gets parsed by the golang templating engine so the comments need to be in that format.

Reply:
will remove the comments, as agreed